portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,45 @@
-#content {
+/*#content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
 #greeting-container {
   margin-top: 20px;
+}*/
+body{   color:rgb(253, 253, 253);
```

Comment:
Add newline after {, same with remaining selectors in this file

Reply:
thank you just fixed it ill go commit it so you can see